Cargo.toml

```diff
@@ -11,7 +11,8 @@ readme = "README.md"
 
 [features]
 flamegraph = ["inferno"]
-protobuf = ["prost", "prost-derive", "prost-build"]
+prost-protobuf = ["pprof-protobuf/prost-protobuf"]
+rust-protobuf = ["pprof-protobuf/rust-protobuf"]
```

Comment:
How about naming it as prost-codec and protobuf-codec just like what TiKV and other projects do.

Reply:
Yes! I will rename them.